website_sale_infinite_scroll/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,2 @@
+To select different preload go to * General Settings > Website:
+ - "Website sale infinite scroll preloader" change preloader;
```

Comment:
To customize preloading icon, go to * General Settings > Website > Website sale infinite scroll preloader"